README.md

```
@@ -449,20 +449,20 @@ Contributions are welcome! Please follow these guidelines:
 
-## 📜 License
+## License
 
 Mozilla Public License 2.0 (MPL-2.0)
 
 ---
 
-## 🔗 Links
+## Links
 
 - **Starknet**: https://starknet.io
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)